modules/thunder_workflow/css/edit-form.css

```diff
@@ -0,0 +1,16 @@
+.form-actions div .form-item {
+  margin: 0;
+}
+
+.form-actions div {
+  margin-top: var(--space-m);
+  margin-bottom: var(--space-m);
+  margin-right: var(--gin-spacing-xs);
+}
+
+.form-actions select {
+  font-weight: var(--gin-font-weight-semibold);
+  color: var(--gin-color-primary);
+  border: 2px solid var(--gin-color-primary) !important;
+  border-radius: var(--gin-border-m);
+}
```

Comment:
I would suggest not trying to style the select as a button, I think it looks better just left alone.